internal/models/amr.go

```diff
@@ -22,7 +22,7 @@ func (AMRClaim) TableName() string {
 }
 
 func (cl *AMRClaim) IsAAL2Claim() bool {
-	return *cl.AuthenticationMethod == TOTPSignIn.String() || *cl.AuthenticationMethod == MFAPhone.String()
```

Comment:
Should find a way to include the `factor_id` or similar in the claims.

Reply:
Does that make a difference?

Reply:
Not really, only for enforcing RLS policies. We can leave out for now and add if requested.